lib/extras/dec/decode.h

```diff
@@ -32,7 +32,8 @@ enum class Codec : uint32_t {
   kPGX,
   kJPG,
   kGIF,
-  kEXR
+  kEXR,
+  kJXL
```

Comment:
Doesn't C++ support final trailing commas? I know C does.

Reply:
Yes it does, not sure what clang-format thinks about it though

Reply:
No problem AFAIK